setup/basic.md

```diff
@@ -358,11 +451,12 @@ IMAGE_PATH is the path to where you want the image saved -- e.g., `~/turtlebot4_
 
 - Get the SD flash script from `turtlebot4_setup` and flash the SD card:
 
+
 ```bash
-wget https://raw.githubusercontent.com/turtlebot/turtlebot4_setup/humble/scripts/sd_flash.sh
+wget https://raw.githubusercontent.com/turtlebot/turtlebot4_setup/jazzy/scripts/sd_flash.sh
```

Comment:
The Jazzy version adds
```bash
for device in $device_names
do
    if [[ $device == mmcblk* ]]; then
        last_partition="${device}p2"
    else
        last_partition="${device}2"
    fi

    sudo growpart /dev/$device 2
    sudo resize2fs /dev/${last_partition}
done
```
to the end of the script to automatically expand the `writable` partition to the maximum space of the SD card. I haven't tested the Jazzy script with Humble images, so I'm not sure if the change is backwards-compatible.  But using the Jazzy version of the script with the Jazzy images is recommended.

Reply:
If we aren't certain it will work with Humble then let's create a Humble tab and Jazzy tab or, I am okay with leaving this as Jazzy if you are going to test and make sure that it works properly.

That being said, was there a change to how Ubuntu 24.04 handles this? I know with 22.04 it automatically expanded to use the full drive on first boot so there was no need to do this manually.

Reply:
I didn't test to see if 24.04 automatically expanded the partition (I admit I didn't know this was a thing 22.04 did either). But a quick test shows that the Humble images aren't compatible with the new script, but the new script is working well with the Jazzy images. So I've reverted to having tabbed commands.